data_request_api/data_request_api/content/dreq_content.py

```diff
@@ -121,9 +121,6 @@ def get_cached(**kwargs):
                 json_export = _json_raw
             elif kwargs["export"] == "release":
                 json_export = _json_release
-            else:
-                warnings.warn(f"Unknown export type '{kwargs['export']}'. Defaulting to 'release'.")
-                json_export = _json_release
```

Comment:
The only downside is, that we cannot default to kwarg values within functions if we do not like the selected value. But that would anyways be a confusing and inconsistent behaviour.